bigcache_test.go

```diff
@@ -1181,7 +1180,7 @@ func TestBigCache_GetWithInfo(t *testing.T) {
 			name:     "Expired",
 			clock:    5,
 			wantData: value,
-			wantResp: Response{},
+			wantResp: Response{EntryStatus: Expired},
```

Comment:
This test case conflicts with the test name - if the name is `Expired` maybe it should be expected to be expired.